### ISSUE TITLE
Downstream tests fail on emulator

### ISSUE THREAD

**Issue by saraedum** [conda-forge/cppyy-feedstock#52]
Testing the e-antic-feedstock, we get on ppc64:
```
import: 'pyeantic'
(Re-)building pre-compiled headers (options: -O2 -mavx); this may take a minute ...
warning: argument unused during compilation: '-mavx'
In file included from input_line_8:1:
./etc/dictpch/allLinkDefs.h:1:10: fatal error: 'etc/dictpch/core/thread/inc/LinkDef.h' file not found
#include "etc/dictpch/core/thread/inc/LinkDef.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Error: ./bin/rootcling: Linkdef compilation failure
```

**Comment by wlav** [comment]
The `-mavx` flag is added if `avx` is in the output of `sysctl machdep.cpu.features`. That `etc/dictpch/core/thread/inc/LinkDef.h` should exist `<site-packages>/cppyy_backend`. I actually don't know how well ppc64 is supported; I haven't had a ppc64 machine in ages. You may want to test once with the `CLING_STANDARD_PCH` envar set to `None`. That is slower on startup, because all headers will be parsed every time, but it will let tests proceed to give a clear idea of what will happen.

**Comment by saraedum** [comment]
The files are simply missing from cppyy-cling it seems on some platforms, see https://github.com/conda-forge/cppyy-cling-feedstock/issues/42.

**Comment by saraedum** [comment]
Fixed in #43. (Sorry, I broke the CMakeLists.txt with a patch for cross-compilation.)
